Documentation/ceph-cluster-crd.md

```diff
@@ -1615,3 +1585,47 @@ data:
 
 Note: if you are using self-signed certificates (not known/approved by a proper CA) you must pass `VAULT_SKIP_VERIFY: true`.
 Communications will remain encrypted but the validity of the certificate will not be verified.
+
+### Deleting a CephCluster
+Deleting a CephCluster an uncommon-in-production but necessary part of a storage cluster's
+lifecycle. It is simple to initiate deletion of a CephCluster using `kubectl delete` just as with
+any Kubernetes resource.
+
+During deletion of a CephCluster resource, Rook protects against accidental or premature destruction
+of user data by blocking deletion if there are any other Rook-Ceph Custom Resources that reference
+the CephCluster being deleted. Rook will warn about which other resources are blocking deletion in
+three ways until all blocking resources are deleted:
+1. An event will be registered on the CephCluster resource
+1. A status condition will be added to the CephCluster resource
+1. An error will be added to the Rook-Ceph Operator log
+#### Cleanup policy
```

Comment:
nit: add a blank line before and after headers. vscode is always complaining about this. :)